neqo-crypto/build.rs

```diff
@@ -295,6 +298,53 @@ fn build_bindings(base: &str, bindings: &Bindings, flags: &[String], gecko: bool
         .expect("couldn't write bindings");
 }
 
+fn pkg_config() -> Vec<String> {
+    let modversion = Command::new("pkg-config")
+        .args(["--modversion", "nss"])
+        .output()
+        .expect("pkg-config reports NSS as absent")
+        .stdout;
+    let modversion = String::from_utf8(modversion).expect("non-UTF8 from pkg-config");
+    let modversion = modversion.trim().to_string();
+    // The NSS version number does not follow semver numbering, because it omits the patch version
+    // when that's 0. Deal with that.
+    let modversion_for_cmp = if modversion.split('.').count() == 2 {
```

Comment:
Rather than split, because you don't need the slices, you can do this:

```suggestion
    let modversion_for_cmp = if modversion.chars().filter(|c| c == '.').count() == 1 {
```